integration-tests/docker/Dockerfile

```diff
@@ -46,6 +47,9 @@ ADD lib/* /usr/local/druid/lib/
 RUN wget -q "https://repo1.maven.org/maven2/mysql/mysql-connector-java/$MYSQL_VERSION/mysql-connector-java-$MYSQL_VERSION.jar" \
     -O /usr/local/druid/lib/mysql-connector-java.jar
 
+RUN wget -q "https://packages.confluent.io/maven/io/confluent/kafka-protobuf-provider/$CONFLUENT_VERSION/kafka-protobuf-provider-$CONFLUENT_VERSION.jar" \
+    -O /usr/local/druid/lib/kafka-protobuf-provider.jar
+
```

Comment:
we should be able to avoid this by enabling the protobuf-extension which already includes this dependency

Reply:
It doesn't actually include the dependency, there are licensing issues and that jar isn't solely Apache licensed, but actually dual Apache and Confluent community license, so people need to fetch it themselves similar to MySQL, see https://github.com/apache/druid/pull/10839#discussion_r570888190

Reply:
good point, I did not realize that. Maybe a comment would be worthwhile